homeassistant/components/sharkiq/strings.json

```diff
@@ -14,7 +20,8 @@
       "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "abort": {
-      "already_configured_account": "[%key:common::config_flow::abort::already_configured_account%]"
+      "already_configured_account": "[%key:common::config_flow::abort::already_configured_account%]",
+      "reauth_successful": "[%key:common::config_flow::data::access_token%] updated successfully"
     }
```

Comment:
We need to add `cannot_connect` and `unknown` abort reasons. They are used in the reauth step.